content/05.methods.md

```diff
@@ -42,13 +42,13 @@ nextflow run nf-core/eager -r 2.1.0 -profile conda --input '/<path>/<to>/*/*R1*.
 
 #### Profiles
 
-We utilize a central configuration profile repository to enable users from
```

Comment:
Most journals insist on American English instead of BE. Should be consistently done everywhere then.

Reply:
I don't care! We are going :gb:

Reply:
:gb:

Reply:
I second James! 🇬🇧